PR 2

CamHargrave opened the pull request.

CamHargrave committed:
Attic Level

Made Recording prefab, greyboxed out attic, along with desk, journal and exit from the attic.

CamHargrave committed:
Changed Scene

Changed Scene name to apropriately reflect what it should be.

CamHargrave committed:
Attic tweeks

Made door a prefab along with journal and made the entire area children under Empty object Attic.

CamHargrave committed:
Next Level done

Made upper level and transition stairs to Main Level.

CamHargrave committed:
Gonna Change my Scenes

Looking into changing the staircase and general UpplerLevel and MainLevel scene

CamHargrave committed:
Finish Upper and Main

Main and Upper level are done

CamHargrave committed:
Added VFX Graph

Didn't really do much here, just added VFX Graph along with some doors for the Main Floor of the house. I have a desk prefab that I went and added a box collider to and textured and untextured face. I also went and changed the rotation of the desk in my two scenes.

CamHargrave committed:
Added Master Room

Added Master Bedroom

CamHargrave committed:
Lots Added

Scripts added in

PatrickIckes commented on Adventure/Assets/Scripts/InteractiveObject.cs:
```diff
@@ -0,0 +1,11 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
```
Add a summary here so someone can understand what this script does.

CamHargrave replied:
Added the summary

PatrickIckes commented on Adventure/Assets/Scripts/RayCast.cs:
```diff
@@ -0,0 +1,35 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
```
Again, add a summary.

CamHargrave replied:
Added summary

PatrickIckes commented on Adventure/Assets/Scripts/RayCast.cs:
```diff
@@ -0,0 +1,35 @@
+
+public class RayCast : MonoBehaviour
+{
+    [Tooltip("Starting point f raycast used to detect interatives.")]
```
Spelling mistake - "interactives"

PatrickIckes replied:
After making all the changes, you're all good for merging.

CamHargrave replied:
fixed the spelling mistake

CamHargrave committed:
LittleMore

get a bit more of the interative object stuff done.

dantogno commented:
@CamHargrave Make these changes so @PatrickIckes can merge your branch!!

CamHargrave committed:
Updated

Handled the Comments in PR 2

CamHargrave committed:
Updates

Door has animation, display text does its job, sound doesn't play for journal, Interacting with objects not working.

CamHargrave committed:
Updates

Added the canvas with reticle and interactive object scripts to second scene
also applied textures(well vertex colors) to all objects with out textures.

CamHargrave committed:
Update

Added Title scene
